plugins/outputs/dynatrace/dynatrace.go

```diff
@@ -236,6 +247,16 @@ func (d *Dynatrace) Init() error {
 		},
 		Timeout: time.Duration(d.Timeout),
 	}
+
+	dims := []dimensions.Dimension{}
+	for key, value := range d.DefaultDimensions {
+		dims = append(dims, dimensions.NewDimension(key, value))
+	}
+	d.normalizedDefaultDimensions = dimensions.NewNormalizedDimensionList(dims...)
+	d.normalizedStaticDimensions = dimensions.MergeLists(
+		dimensions.NewNormalizedDimensionList(dimensions.NewDimension("dt.metrics.source", "telegraf")),
+	)
```

Comment:
`MergeLists` is not necessary now that the other list was removed.

Reply:
I agree here.